Packs/GCP/Integrations/GCP/GCP.yml

```diff
@@ -141,7 +141,438 @@ script:
+      type: String
+    - contextPath: GCP.Compute.Snapshot.sourceDiskId
+      description: Unique ID of the source disk used to create the snapshot.
+      type: String
+    - contextPath: GCP.Compute.Snapshot.sourceSnapshotSchedulePolicy
+      description: URL of the snapshot schedule policy used to create this snapshot.
+      type: String
+    - contextPath: GCP.Compute.Snapshot.sourceSnapshotSchedulePolicyId
+      description: Unique ID of the snapshot schedule policy used to create this snapshot.
+      type: String
+    - contextPath: GCP.Compute.Snapshot.storageBytes
+      description: Total storage size of the snapshot in bytes.
+      type: Number
+    - contextPath: GCP.Compute.Snapshot.storageBytesStatus
+      description: Status of the storage bytes usage, for example UP_TO_DATE.
+      type: String
+    - contextPath: GCP.Compute.Snapshot.storageLocations
+      description: List of storage locations for the snapshot.
+      type: Unknown
+  - name: gcp-compute-instances-aggregated-list-by-ip
+    description: Aggregated list of instances across all zones; can be filtered by internal or external IP.
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: ip_address
+      required: true
+      description: The IP address to search for.
+    - name: match_external
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      description: If true, match against external NAT IPs; otherwise internal NIC IPs.
+    - name: limit
+      description: Maximum number of results to return. Acceptable values are 0 to 500, inclusive.
+    - name: page_token
+      description: Token for pagination.
+    outputs:
+    - contextPath: GCP.Compute.Instance.name
+      description: Instance name.
+      type: string
+    - contextPath: GCP.Compute.Instance.id
+      description: Instance ID.
+      type: string
+    - contextPath: GCP.Compute.Instance.zone
+      description: Instance zone URL.
+      type: string
+    - contextPath: GCP.Compute.Instance.status
+      description: Instance status.
+      type: string
+    - contextPath: GCP.Compute.Instance.networkInterfaces
+      description: Network interfaces of the instance.
+      type: Unknown
+  - name: gcp-compute-network-tag-set
+    description: Adds a network tag to a VM instance (merges with existing tags).
+    arguments:
+    - name: project_id
+      required: true
+      description: GCP project ID.
+    - name: zone
+      required: true
+      description: Zone of the VM (e.g., us-central1-a).
+    - name: resource_name
+      required: true
+      description: Instance name.
+    - name: tag
+      required: true
+      description: Tag to add.
+    - name: tags_fingerprint
+      description: Fingerprint of the previous set of tags for this resource, used to prevent conflicts. Provide the latest fingerprint value when making a request to add or change tags.
```

Comment:
```suggestion
      description: Fingerprint of the previous set of tags for this resource, used to prevent conflicts. Provide the latest fingerprint value when making a request to add or change tags. To retrieve the finger print use the command gcp-compute-instance-get.
```